strategy/src/lib.rs

```diff
@@ -0,0 +1,391 @@
+// SPDX-Licenser-Identifier: MIT OR Apache-2.0
+
+//! A strategy for using the [`event-listener`] crate in both blocking
```

Comment:
nit: the limit is 100 chars (74 is for git commits :)) and best to make use of it to reduce LOCs.